corn/simpleBlogHomePage/index.html

```diff
@@ -1,10 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <title>Simple Blog Home Page</title>
-    <link rel="stylesheet" href="styles.css" />
-  </head>
-  <body>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Simple Blog Home Page</title>
+  <link rel="stylesheet" href="styles.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Kanit:wght@900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header-padding">
+    <a href="./"><img src="./img/logo.png" alt="dasdas" height="80px" width="80px"></a>
+    <div class="social">
+      <a href="https://twitter.com/"><img src="./img/twitter.png" alt="twitter" height="24px" width="24px"></a>
+      &dot;
+      <a href="https://dribbble.com/"><img src="./img/dribbble.png" alt="dribbble" height="24px" width="24px"></a>
+      &dot;
+      <a href="https://figma.com/"><img src="./img/figma.png" alt="figma" height="24px" width="24px"></a>
+      &dot;
+      <a href="github.com"><img src="./img/github.png" alt="github" height="24px" width="24px"></a>
+    </div>
```

Comment:
```suggestion
    <a href="./"><img src="./img/logo.png" alt="dasdas"></a>
    <div class="social">
      <a href="https://twitter.com/"><img src="./img/twitter.png" alt="twitter"></a>
      &dot;
      <a href="https://dribbble.com/"><img src="./img/dribbble.png" alt="dribbble"></a>
      &dot;
      <a href="https://figma.com/"><img src="./img/figma.png" alt="figma"></a>
      &dot;
      <a href="github.com"><img src="./img/github.png" alt="github"></a>
    </div>
```
because icons are blurred